src/App.jsx

```diff
@@ -14,70 +15,67 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [food, setFood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {food
+          ? (
+            <>
+              {`${food} is selected`}
+              <button
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+                onClick={() => setFood('')}
+              />
+            </>
+          )
+          : (
+            'No goods selected'
+          )}
+      </h1>
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+      {goods.map(good => (
+        <table className="table">
+          <tbody>
+            <tr data-cy="Good" className={`${food === good ? 'has-background-success-light' : ''}`}>
+              {food === good
+                ? (
+                  <td>
+                    <button
+                      data-cy="RemoveButton"
+                      type="button"
+                      className="button is-info"
+                      onClick={() => setFood('')}
+                    >
+                      -
+                    </button>
+                  </td>
+                )
+                : (
+                  <td>
+                    <button
+                      data-cy="AddButton"
+                      type="button"
+                      className="button"
+                      onClick={() => setFood(good)}
+                    >
+                      +
+                    </button>
```

Comment:
You could also avoid repeating the same element
```suggestion
                   <button
                  data-cy={food === good
                    ? 'RemoveButton'
                    : 'AddButton'
                  }
                  type="button"
                  className={`button ${food === good && 'is-info'}`}
                  onClick={() => setFood(prevFood => (prevFood === good ? '' : good))}
                >
                  {food === good ? '-' : '+'}
                </button>
```
and would be nice to move the logic `setFood(prevFood => (prevFood === good ? '' : good))` in some separate handler function